scipy/optimize/optimize.py

```diff
@@ -1465,7 +1465,8 @@ def _minimize_newtoncg(fun, x0, args=(), jac=None, hess=None, hessp=None,
             A = fhess(*(xk,) + args)
             hcalls = hcalls + 1
 
-        while numpy.add.reduce(numpy.abs(ri)) > termcond:
+        k2 = 0
```

Comment:
I realize that there are probably many places to improve clarity in this code. Buy maybe name `k2` as `cg_iter` or something like this.